docs/source/tutorials/playing-command.rst

```diff
@@ -0,0 +1,129 @@
+one is the ``bot``, i.e. an instance of Sopel (wrapped for your convenience)
+that you can use to :doc:`/plugin/bot`.
+
+In the previous tutorial, we used ``bot.reply``, which is convenient but not
+always what you want. Maybe you want the bot to say this::
+
+    <YourNick> .hello
+    <Sopel> Hello YourNick, have a nice day!
+
+For that, you need the second argument: the ``trigger``. It is an object with
+all the information you need about the message that triggered the callable,
+such as the message itself, the channel, the type of message, etc. and what we
+need for now is the :attr:`trigger.nick <sopel.trigger.Trigger.nick>`
+attribute::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Say Hello <user>, have a nice day!"""
+        bot.say(f'Hello {trigger.nick}, have a nice day!')
+
+.. seealso::
+
+    You can learn much more about the :class:`trigger <sopel.trigger.Trigger>`
+    object by reading its documentation.
+
+Command with arguments
+======================
+
+The trigger object can do much more for you: if a user add arguments to the
+command, like this: ``.hello morning``, you can detect and use that argument::
+
+    from sopel import plugin
+
+    @plugin.command('hello')
+    def hello(bot, trigger):
+        """Say Hello <user>, have a nice day!"""
+        # groups 3 to 6 are the first, second, third, and fourth command arg
+        when = trigger.group(3)
+        # select a different greeting depending on when
+        greeting = {
+            'morning': 'and good morning!',
+            'noon': 'are you having lunch?',
+            'night': 'I hope it was a good day!',
+            'evening': 'good evening to you!'
+        }.get(when, 'have a nice day!')  # default to "nice day"
+        # say hello
+        bot.say(f'Hello {trigger.nick}, {greeting}')
+
+Now the command will be able to react a bit more to your user::
+
+    <YourNick> .hello morning
+    <Sopel> Hello YourNick, and good morning!
+    <YourNick> .hello noon
+    <Sopel> Hello YourNick, are you having lunch?
+
+.. seealso::
+
+    You can learn much more about the :class:`~sopel.plugin.command` decorator
+    by reading its documentation.
+
+And... action!
+==============
+
+Some users say ``.hello`` out loud, and other will say it with an action. How
+do you react to these? Let's go back to our initial example of the ``shrug``
+command: it uses a second decorator, ``action_command('shrugs')``, with a
+different name. How does that work?
+
+Sopel knows how to register the same plugin callable for different types of
+trigger, so both ``.shrug`` and ``/me shrugs`` work. For example, you could do
+that for your hello plugin::
+
+    @plugin.command('hello')
+    @plugin.action_command('hi')
+    def hello(bot, trigger):
+        ...
+
+And so, in chat, you will see that::
+
+    <YourNick> .hello
+    <Sopel> Hello YourNick, have a nice day!
+    *YourNick hi*
```

Comment:
Is this the action format for any actual clients? Usually I see something like this:

```suggestion
    * YourNick hi
```